aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -100,7 +100,52 @@ For more information, see [dotnet-counters](/dotnet/core/diagnostics/dotnet-coun
 
 ## Create custom metrics
 
-See [Create custom metrics](/dotnet/core/diagnostics/metrics#create-custom-metrics) for information on creating custom metrics.
+Metrics are created using APIs in the <xref:System.Diagnostics.Metrics> namespace. See [Create custom metrics](/dotnet/core/diagnostics/metrics-instrumentation#create-custom-metrics) for information on creating custom metrics.
+
+### Creating metrics in ASP.NET Core apps with `IMeterFactory`
+
+It's recommended to create `Meter` instances in ASP.NET Core apps with <xref:System.Diagnostics.Metrics.IMeterFactory>.
```

Comment:
Select one of these suggestions, ordered by my favorite first.

Folks that are smart enough to use alternative approaches are smart enough this is not an absolute directive.

```suggestion
Create `Meter` instances in ASP.NET Core apps with <xref:System.Diagnostics.Metrics.IMeterFactory>.
```

```suggestion
`Meter` instances in ASP.NET Core apps should generally be created with <xref:System.Diagnostics.Metrics.IMeterFactory>.
```

```suggestion
We recommended creating `Meter` instances in ASP.NET Core apps with <xref:System.Diagnostics.Metrics.IMeterFactory>.
```